tests/CMakeLists.txt

```diff
@@ -7,15 +7,15 @@ if(${SOURCE_FORMAT_ENABLED})
     add_test(NAME format WORKING_DIRECTORY ${CMAKE_BINARY_DIR} COMMAND cmake --build ${CMAKE_BINARY_DIR} --target format-check)
 endif()
 
-# list of all the tests in each directory
-set(tests test_io test_fd_comm test_init_destroy_client test_init_destroy_server test_client_thread test_thread_messages)
+# list of all the tests that don't require SSH and TLS
+set(tests test_unix_socket test_client_thread test_fd_comm test_init_destroy_client test_init_destroy_server test_io test_thread_messages)
 
 # only enable PAM tests if the version of PAM is greater than 1.4
 if(LIBPAM_HAVE_CONFDIR)
-    list(APPEND tests test_pam)
+    list(APPEND tests test_auth)
 endif()
 
-set(client_tests test_client test_client_messages)
+# set(client_tests test_client test_client_messages)
```

Comment:
Meaningful client tests may be kept but should be reorganized and part of the main test suite.